ush/python/pygfs/task/aero_analysis.py

```diff
@@ -171,12 +194,13 @@ def finalize(self: Analysis) -> None:
 
         # ---- move increments to ROTDIR
         logger.info('Moving increments to ROTDIR')
-        fms_inc_file_template = os.path.join(self.task_config['DATA'], 'anl', f'aeroinc.{self.task_config.cdate_fv3}.fv_tracer.res.tileX.nc')
+        template = 'aeroinc.{}.fv_tracer.res.tile{}.nc'.format(to_fv3time(self.task_config.current_cycle), '{tilenum}')
```

Comment:
```suggestion
        template = f'aeroinc.{to_fv3time(self.task_config.current_cycle)}.fv_tracer.res.tile{{tilenum}}'
```

The double-curlies come out as a literal `{tilenum}` for use with format later.